src/main/java/org/fcrepo/auth/webac/LinkHeaderProvider.java

```diff
@@ -31,6 +34,7 @@
 import org.fcrepo.kernel.modeshape.rdf.impl.DefaultIdentifierTranslator;
 import org.fcrepo.kernel.modeshape.rdf.impl.PropertiesRdfContext;
 
+import org.apache.commons.lang3.tuple.Pair;
```

Comment:
No, it's not part of the task in hand. Can you make a ticket and assign it to me? Something like "Replace use of Pair with a type with specific semantics." or something like that.